verifact/forum/models.py

```diff
@@ -10,3 +11,22 @@ class Question(models.Model):
 
     def __str__(self):
         return self.text
+
+
+class Answer(models.Model):
+    class AnswerTypes(models.IntegerChoices):
+        TRUE = 1, _("True")
+        FALSE = 2, _("False")
+        OTHER = 3, _("Other")
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    answer = models.IntegerField(choices=AnswerTypes.choices)
+    text = models.CharField(max_length=2048)
+    citation_url = models.CharField(max_length=2048)
+    citation_title = models.CharField(max_length=2048)
+    credible_count = models.IntegerField()
+    not_credible_count = models.IntegerField()
+    question_id = models.ForeignKey(Question, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return "[%s] %s" % (self.get_answer_display(), self.text)
```

Comment:
Would it be better to return a human readable answer here? Seems like double work to have the front-end map these again.

Reply:
Depending on what we decide here, we may be able to provide a simpler tuple instead of the `IntegerChoices`. Either works though.